frontend/src/metabase-types/api/alert.ts

```diff
@@ -0,0 +1,5 @@
+export type AlertId = number;
+
+export interface Alert {
```

Comment:
we have already Alert type in scope of my branch

Reply:
@uladzimirdev would that be ok if we merge this branch first to unblock @oisincoveney and you could add other alert fields in your PR?

Reply:
I'm waiting for one test job to pass, so most likely my PR will be a bit faster to merge

Reply:
merged master into your branch

Reply:
Thanks!